addons/mail/static/src/webclient/commands/mail_providers.js

```diff
@@ -0,0 +1,67 @@
+/** @odoo-module */
+
+import { registry } from "@web/core/registry";
+
+// TODO LEGACY ENV
+import env from "web.env";
+
+const { Component } = owl;
+const { xml } = owl.tags;
+
+class DialogCommand extends Component {}
+DialogCommand.template = xml`
+    <t t-name="web.dialogCommand" owl="1">
```

Comment:
Isn't this `<t>` tag useless when giving the template directly?
`t-name` shouldn't be necessary, and `owl="1"` is only used during templates loading from assets.